django_x509/base/models.py

```diff
@@ -32,6 +32,14 @@
     ('sha256', 'SHA256'),
     ('sha384', 'SHA384'),
     ('sha512', 'SHA512'),
+    ('ecdsa-with-sha1', 'ECDSA with SHA1'),
+    ('ecdsa-with-sha256', 'ECDSA with SHA256'),
+    ('ecdsa-with-sha384', 'ECDSA with SHA384'),
+    ('ecdsa-with-sha512', 'ECDSA with SHA512'),
+    ('dsaWithSHA1', 'DSA with SHA1'),
+    ('dsaWithSHA256', 'DSA with SHA256'),
+    ('ed25519', 'Ed25519'),
+    ('ed448', 'Ed448'),
```

Comment:
I don't think it makes sense to alter the digest choices. Try creating a new certificate or CA with any of these and watch it fail.